frontend/test/__support__/cypress.js

```diff
@@ -304,3 +304,12 @@ function addQADatabase(engine, db_display_name, port) {
     expect(status).to.equal(200);
   });
 }
+
+export function visitQuestionAdhoc(question) {
+  if (question.display) {
+    // without "locking" the display, the QB will run its picking logic and override the setting
+    question = Object.assign({}, question, { displayIsLocked: true });
```

Comment:
TIL that we have `displayIsLocked` :)

Reply:
It's meant to be an internal flag signifying that we shouldn't update the query as data changes, but here that leaks a bit since we're creating this URL outside the app.